build.gradle

```diff
@@ -9,10 +9,13 @@ buildscript {
     repositories {
         mavenLocal()
         mavenCentral()
+
+        // Needed for the Android Gradle Plugin to work
         google()
+        jcenter()
```

Comment:
[JCenter is dying :(](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)
What is hosted there and can it be found somewhere else?

Reply:
I know that it's going away but unfortunately the version of the Android Gradle Plugin used depends on it. The build failed with the following error until I added the repostiory.
```error
A problem occurred configuring project ':android'.
> Could not resolve all artifacts for configuration ':android:classpath'.
   > Could not find org.jetbrains.trove4j:trove4j:20160824.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
       - https://dl.google.com/dl/android/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
     Required by:
         project :android > com.android.tools.build:gradle:4.0.2 > com.android.tools.build:builder:4.0.2 > com.android.tools:sdk-common:27.0.2

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```
This is a known issue, unfortunately (see [here](https://youtrack.jetbrains.com/issue/KT-44730), [here](https://youtrack.jetbrains.com/issue/IDEA-261387) and [here](https://stackoverflow.com/q/66049196)). I think the issue should be addressed in a future release of the Android Gradle Plugin but I tried this with the latest version 4.1.2 and that showed the same error. I'm not using versions >= 4.1.0 at the moment though due to other build issues that occur later-on. I am not aware of a better solution for that issue at the moment.

Reply:
Cross-mentioning https://github.com/MovingBlocks/Terasology/issues/4582 for reference here.